flang/include/flang/Lower/ConvertCall.h

```diff
@@ -24,6 +24,10 @@
 
 namespace Fortran::lower {
 
+struct LoweredResult {
+  std::variant<fir::ExtendedValue, hlfir::EntityWithAttributes> result;
+};
```

Comment:
nit: Could this be simplified to
```c++
using LoweredResult = std::variant<fir::ExtendedValue, hlfir::EntityWithAttributes> result;
```

This way callers wouldn't have to add a `.result`.

Reply:
Updated.